src/wp-includes/class-wp-block-patterns-registry.php

```diff
@@ -164,12 +164,16 @@ public function unregister( $pattern_name ) {
 	 */
 	private function prepare_content( $pattern, $hooked_blocks ) {
 		$content = $pattern['content'];
+
+		$before_block_visitor = '_inject_theme_attribute_in_template_part_block';
+		$after_block_visitor  = null;
+		$hooked_blocks        = get_hooked_blocks();
```

Comment:
No need for this line; it would negatively impact performance, so we should remove it.